fastlane/lib/fastlane/environment_printer.rb

```diff
@@ -270,6 +270,10 @@ def self.print_fastlane_files
       env_output
     end
 
+    def self.anonymized_path(path)
+      return ENV['HOME'] ? path.gsub(/^#{ENV['HOME']}/, '~') : path
```

Comment:
In the super edgy edge case that the user's home dir is the prefix of other folders this is gonna output garbage. Example: `$HOME=/work`, `anonymized_path("/workspace/whatever/fastlane")` => `~space/whatever/fastlane`. 

Might be more applicable to linux where the user's home dir is not generally inside the `/Users/` folder.